_data/tasks.yml

```diff
@@ -1,15 +1,15 @@
-bigheader: "Tasks"
-abstract: "Step-by-step instructions for performing operations with Kubernetes."
+bigheader:   "Tasks"
```

Comment:
Oops, sorry, didn't notice this. It's due to an Atom package that automatically formats YAML files.